miden-lib/src/transaction/procedures.rs

```diff
@@ -0,0 +1,239 @@
+use alloc::vec::Vec;
+
+use miden_objects::{Digest, Felt, Hasher};
+
+use super::TransactionKernel;
+
+macro_rules! digest {
+    ($a:expr, $b:expr, $c:expr, $d:expr) => {
+        Digest::new([Felt::new($a), Felt::new($b), Felt::new($c), Felt::new($d)])
+    };
+}
```

Comment:
We don't have a `utils` module in the `miden-objects` crate, should I create it? 

I think moving this macro to a separate file, created just for it is a bit overkill, probably we can do that in case this macro will be used somewhere else?

Reply:
We do have the [utils](https://github.com/0xPolygonMiden/miden-base/blob/next/objects/src/lib.rs#L46) module - it is just mostly re-exports. We can add this macro there.

Reply:
Ah, I see, I was looking for the `utils` file, but we have the module inside the `lib.rs` file. Sure, I'll move it there.